### DISCUSSION TITLE
Data selection conditions

### DISCUSSION THREAD

**Discussion by bit-config** [prisma/prisma#8656] [Q&A] [answered]
I am using postgresql as a database.
In it I can make a request
SELECT * FROM public.client Where id IN (1,2)
It will output all the data. But how can this be done in prisma?

```ts
await prisma.post.findMany({
            where: { 
                id: [1,2]
            },
        })

```
_schema.prisma_
```prisma
model post{
  id   Int    @id @default(autoincrement())
  name String
}

```

**Comment by ryands17** [comment, marked as answer]
@4cfg 👋 

Using the `in` operator: https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#in

**Reply by bit-config** [reply]
Thanks. It is strange that through a search on the site I could not find this.

**Reply by ryands17** [reply]
The [reference](https://www.prisma.io/docs/reference) will contain the API of everything needed including `PrismaClient` 🙂

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
